draco/analysis/sensitivity.py

```diff
@@ -0,0 +1,252 @@
+# === Start Python 2/3 compatibility
+from __future__ import absolute_import, division, print_function, unicode_literals
+from future.builtins import *  # noqa  pylint: disable=W0401, W0614
+from future.builtins.disabled import *  # noqa  pylint: disable=W0401, W0614
+
+# === End Python 2/3 compatibility
+
+import numpy as np
+
+from caput import config
+
+from ..core import task, io, containers
+from ..util import tools
+
+
+class ComputeSystemSensitivity(task.SingleTask):
+    """Compute the sensitivity of beamformed visibilities.
+
+    Parameters
+    ----------
+    exclude_intracyl : bool
+        Exclude the intracylinder baselines in the sensitivity estimate.
+        Default is to use all baselines.  Note that a RuntimeError
+        will be raised if exclude_intracyl is True and the visibilities
+        have already been stacked over cylinder.
+    """
+
+    exclude_intracyl = config.Property(proptype=bool, default=False)
```

Comment:
This doesn't completely make sense in a generic context, but we can figure out what (if anything) to do about it later.

Reply:
Yeah, we originally had a cut on baseline distance for that reason.  But we don't have the freedom to make arbitrary baseline cuts on the stacked data and still do the exact radiometer estimate, so it was changed to this.